io.github.obsidian-level-maker.metainfo.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>io.github.obsidian_level_maker</id>
+  
+  <name>Obsidian Level Maker</name>
+  <summary>Map generator for 90s FPS games</summary>
+  
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0-only</project_license>
+  
+  <description>
+    <p>
+      Obsidian Level Maker is a community continuation of OBLIGE Level Maker, originally by Andrew Apted. It generates random levels that are (hopefully) fun to play. The following games are currently supported: - Doom 1/The Ultimate Doom/Freedoom Phase 1 - Doom 2/Freedoom Phase 2 - Final Doom - Heretic - Hexen - Chex Quest 3 - HacX 1.2 - Harmony - Strife - Wolfenstein 3-D - Spear of Destiny - Super Noah&apos;s Ark 3-D
+    </p>
+  </description>
+  
+  <launchable type="desktop-id">io.github.obsidian-level-maker.desktop</launchable>
+</component>
```

Comment:
missing mandatoy `url`, `content_rating`, `screenshots`, `releases`.

Reply:
I used the recommended metadata generator. Where is it documented that these are mandatory?